puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testKatello.groovy

```diff
@@ -101,6 +101,7 @@ pipeline {
                     steps {
                         sh "npm install"
```

Comment:
this is still needed, because bastion, correct?

Reply:
I'm not 100% sure it is, but i think it'd is because of the packages.json in the katello dir related to the react stuff there